helm-charts/yunikorn/values.yaml

```diff
@@ -16,7 +16,7 @@
 
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
-
+installCRD: true
```

Comment:
I would disable it by default. Unless until the full feature is implemented. After that we can think about enabling it by default.

Reply:
make sense to me. done.